tests/setup_test.go

```diff
@@ -64,6 +64,8 @@ const (
 	giteaUser02 = "user02"
 
 	agolaUser01 = "user01"
+
+	agolaOrg01 = "org01"
```

Comment:
If you add a const here you should also update all the code to use this const. Probably a new PR after or before this will be better.